libs/blocks/chart/chart.js

```diff
@@ -44,10 +44,34 @@ function processDataset(data) {
   return dataset;
 }
 
-function processSeries(data) {
-  const series = [];
-  // TODO: Series data
-  return series;
+const parseValue = (value) => (Number.isInteger(Number(value)) ? parseInt(value, 10) : value);
+
+export function processMarkData(series) {
+  const seriesOptions = series.reduce((acc, mark) => {
+    acc[mark.Type] ??= { data: [] };
```

Comment:
According to airbnb style guide, I believe this is how the formatting should be.
```suggestion
    acc[mark.Type] ??= { data: [] };
    
    const markData = acc[mark.Type].data;
    const split = mark.Value?.split('-');
    const value = parseValue(split[0]);
```

Reply:
@sanjayms01 Are you saying there needs to be a blank line before the `const`s so they are grouped?

Reply:
Yeah, that's correct.

Reply:
Fixed.